### PR TITLE
[#XGB-97] Add section Operations Security

### PR DESCRIPTION
**Pull request by raphiz**
See also: https://github.com/xmpp-grid-broker/xmpp-grid-broker/pull/4

### CODE REVIEW

File: documents/final-submission-document/chapters/3_concept.tex [hunk 1]
```diff
@@ -259,7 +258,25 @@ \subsection{Client Security Checklist}
...
+    \item User input is always escaped using the mechanisms provided by the framework (eg. Angular Forms)
     \item XMPP-Messages are validated to contain only the specified result-types
-\end{itemize}
\ No newline at end of file
+\end{itemize}
+
+\subsection{Operations Security}
+
+
+To mitigate some risks administrators must configure the surrounding systems correctly.
```
Comment (fabianhauser) [inline documents/final-submission-document/chapters/3_concept.tex]: Should there be a comma after risks?

Comment (fabianhauser) [reply documents/final-submission-document/chapters/3_concept.tex]: That's also easier to read, so I changed it 👍

**Comment by fabianhauser** [comment]
@raphiz I rewrote the sentences a bit, please take a look if you agree 🙂

**Comment by raphiz** [comment]
As discussed: Add footnote to mention that AR happend

**Comment by raphiz** [comment]
@fabianhauser nice, I added the AR footnote and will merge when the build is done :+1:

**Pull request merged**